src/test/rustdoc-gui/source-code-page.goml

```diff
@@ -11,3 +11,5 @@ assert-attribute: (".line-numbers > span:nth-child(4)", {"class": "line-highligh
 assert-attribute: (".line-numbers > span:nth-child(5)", {"class": "line-highlighted"})
 assert-attribute: (".line-numbers > span:nth-child(6)", {"class": "line-highlighted"})
 assert-attribute-false: (".line-numbers > span:nth-child(7)", {"class": "line-highlighted"})
+// This is to ensure that the content is correctly align with the line numbers.
+compare-elements-position: ("//*[@id='1']", ".rust > span", ("y"))
```

Comment:
This doesn't hurt, but I also don't think it helps very much? Usually the isuses come from less common browsers because the line numbers use a metric ton of CSS to allow being out-of-line with the code itself.

Reply:
Not at all. The issue appeared at least three times on all browsers. There were the line numbers and then completely below the source code. It was caused by changes on the CSS as you may have guessed. ;)